apps/web/ee/pages/api/integrations/stripepayment/webhook.ts

```diff
@@ -174,6 +174,10 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
 
     const event = stripe.webhooks.constructEvent(payload, sig, process.env.STRIPE_WEBHOOK_SECRET);
 
+    if (event.account) {
```

Comment:
From Stripe doc: In addition to the documented response properties in the [event object reference](https://stripe.com/docs/api#events), each event for a connected account will also contain a top-level account property. It identifies the user for whom the webhook is being sent.

This should allow our webhook to return 200 and not affect our self hosted user's webhooks.

Reply:
Shouldn't we do it only if payment entry is not there? 

on cal.com if a booker pays organizer, payment entry should be there